bitnami/cassandra/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: cassandra
 sources:
   - https://github.com/bitnami/bitnami-docker-cassandra
   - http://cassandra.apache.org
-version: 7.3.3
+version: 7.3.4
```

Comment:
this change adds a new feature could you increase the version to 7.4.0?

Reply:
Done